utils/git-sync-deps

```diff
@@ -60,217 +60,218 @@ from builtins import bytes
 
```

Comment:
This file is imported from the skia repo. Can we leave it un-formatted so it's easier to see differences if we update the script?

Reply:
Done